### ISSUE TITLE
Tests do not work with phantomjs 2.0

### ISSUE THREAD

**Issue by lambdarocket** [Day8/re-frame-template#10]
Tests do not run with the newest phantomjs. Phantomjs deprecated args tuple - we cannot use it anymore.

**Comment by mll** [comment]
Yeah, I see this too.

**Comment by gadfly361** [comment]
@lambdarocket Thanks for mentioning this!  I will try to look into it.

---
Notes to self:

From [cljs.test blog post](https://nvbn.github.io/2015/06/08/cljs-test/) comment
 
```
FYI the test.js in your example project won't work with PhantomJS v2.0, only with v1.9

To make it work with v2.0, replace the line:
var url = phantom.args[0];
with:
var url = require('system').args[1];
```

Try replacing this [line](https://github.com/Day8/re-frame-template/blob/master/src/leiningen/new/re_frame/test/unit-test.js#L2) with recommendation from blog post comment.

**Comment by lambdarocket** [comment]
Thanks!!!

Best regards, 

Marek Lipert


Dnia 26.09.2015 o godz. 19:31 Matthew Jaoudi <notifications@github.com> napisał(a):

**Comment by danielcompton** [comment]
Take a look at https://github.com/emezeske/lein-cljsbuild/pull/415 for hints on setup to support Phantom 1 and 2 at the same time.

**Comment by gadfly361** [comment]
Just pushed a new version to clojars with fix recommended by @danielcompton (thanks!!)

Before this gets closed, can someone confirm phantomjs 2.0 works for them? @lambdarocket @mll

**Comment by mll** [comment]
@gadfly361 Works great, thanks!
